### PR TITLE
fix compiler error on ubuntu 12.04 x86_64 and remove duplicated items for smartDeviceLink.ini

### PR DESCRIPTION
**Pull request by chenbd**

### CODE REVIEW

File: src/components/transport_manager/src/tcp/tcp_client_listener.cc [hunk 1]
```diff
@@ -42,7 +42,8 @@
 #include <sys/sysctl.h>
 #include <sys/socket.h>
 #ifdef __linux__
-#  include <linux/tcp.h>
+//#  include <linux/tcp.h>
```
Comment (lea255ace) [inline src/components/transport_manager/src/tcp/tcp_client_listener.cc]: Is there a reason to commit commented code into the repository?

Comment (chenbd) [inline src/components/transport_manager/src/tcp/tcp_client_listener.cc]: commented code has been removed.

**Comment by justinjdickow** [comment]
@chenbd I'm going to be merging in a (hopefully) final internal release today or tomorrow before we move completely to pull requests. Just a heads up because it _might_ require a rebase. This change lgtm though

**Comment by chenbd** [comment]
@justinjdickow my branch has been rebased. seems we still have a lot work to do.
where does your internal release code come from? genivi code base?

**Pull request merged**